com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/Ecl.xtext

```diff
@@ -297,6 +309,16 @@ CaseSignificanceFilter:
 FilterValue:
 	FilteredExpressionConstraint | EclConceptReferenceSet;
 
+// History Supplement from ECL 2.0
+Supplement:
+	HistorySupplement;
+
+HistorySupplement:
+	DOUBLE_CURLY_OPEN PLUS HISTORY_KEYWORD {HistorySupplement} history=(HistoryProfile | NestedExpression)? DOUBLE_CURLY_CLOSE;
```

Comment:
Have we turned off `WS` hiding by default? I don't know whether the grammar allows eg. `HISTORY - MIN` or not. 🤔

Reply:
See the top line of the grammar definition: `hidden(WS, ML_COMMENT, SL_COMMENT)`
You can put whitespace and comment basically everywhere.

Reply:
We temporarily un-hide `WS` for unquoted search terms, I think.

Reply:
Fixed